### PR TITLE
#631- Typescript definitions edits: element.stop, container.polyline(), select(): Set, animation.scale()

### PR DESCRIPTION
**Pull request by showdep**

### CODE REVIEW

File: svg.js.d.ts [hunk 1]
```diff
@@ -951,7 +952,7 @@ declare namespace svgjs {
 
         rotate(degrees: number, cx?: number, cy?: number): Animation
         skew(x: number, y: number): Animation
-        scale(x: number, y: number): Animation
+        scale(scaleX: number, scaleY: number, cx?: number, cy?: number): Animation
```
Comment (Fuzzyma) [inline svg.js.d.ts]: `scale` may called with one parameter only. So `scaleY` has to be optional, too. Please change that bit and I guess we can merge that. Thanks for your contribution!

**Comment by dotnetCarpenter** [comment]
@showdep I can not push to this PR. Have you unchecked the option?

Anyway. the line `scale(scaleX: number, scaleY: number, cx?: number, cy?: number): Animation` just have to be `scale(scaleX: number, scaleY?: number, cx?: number, cy?: number): Animation`. Then we are all good.

**Comment by Fuzzyma** [comment]
@dotnetCarpenter as far as i know you cant just push to a repository which is owned by someone else if he didnt add you as maintainer explicitely. And thats not often done.

However I agree with the change proposed by @dotnetCarpenter. We can merge that after

**Comment by Fuzzyma** [comment]
@dotnetCarpenter @showdep I allowed myself to do some changes to the ts file. Please doublecheck so we can merge

**Comment by showdep** [comment]
@Fuzzyma 

> I allowed myself to do some changes to the ts file.
> Please doublecheck so we can merge
Fine with me: param names and optional params

**Comment by Fuzzyma** [comment]
@showdep I updated the definitions again. Can you pls check the current master? I have no typescript environment to check if it works. Would be cool if at least the syntax should be correct everywhere :)

**Pull request merged**